.codesandbox/ci.json

```diff
@@ -5,7 +5,6 @@
   "packages": [
     "packages/markdown",
     "packages/mui-babel-macros",
-    "packages/mui-base",
```

Comment:
Wouldn't this break the Base UI demos? Why was the change made?

Reply:
Marking base as private doesn't allow codesandbox to publish the package.

Reply:
Let's revert the change to make it private.